src/libs/decoders/opus.cpp

```diff
@@ -179,7 +179,7 @@ static opus_int64 RWops_opus_tell(void * stream)
 
     const int64_t current_offset = SDL_RWtell(static_cast<SDL_RWops*>(stream));
     SNDDBG(("Opus ops tell:          "
-            "%ld\n", current_offset));
+            "%ld\n", static_cast<long int>(current_offset)));
```

Comment:
On Windows `long int` is the same as `int` - I doubt it will cause problems, but I you want to stay on the safe side, use `PRIu64` instead of `ld` and avoid down-casting in this case.

Reply:
Thanks; this worked as you predicted!